docs/user/gateways.md

```diff
@@ -37,6 +37,12 @@ The Telemetry gateways automatically enrich your data by adding the following at
   - Deployment/DaemonSet/StatefulSet/Job name
   - Namespace
   - Cluster name
+- Cloud provider attributes: The gateway automatically adds [cloud provider](https://opentelemetry.io/docs/specs/semconv/resource/cloud/) attributes to the telemetry data when data is available. The attribute values are mostly based on the well-known Kubernetes labels available on the nodes.
+  - `cloud.provider`: The cloud provider name.
+  - `cloud.region`: The region where the node runs. The value is retrieved from node label `topology.kubernetes.io/region`.
+  - `cloud.availability_zone`: The zone where the node runs. The value is retrieved from node label `topology.kubernetes.io/zone`.
+  - `host.type`: The machine type of the node. The value is retrieved from node label `node.kubernetes.io/instance-type`.
```

Comment:
Maybe have another root entry for the host attributes (that is not related to cloud.) and reference: https://opentelemetry.io/docs/specs/semconv/resource/host/

Reply:
host attributes moved to own entry